### DISCUSSION TITLE
How to use the same token for every script execution?

### DISCUSSION THREAD

**Discussion by wiuxs** [O365/python-o365#1035] [Q&A]
I have a script that performs a check_status of websites, the return is sent by email.

This script will run via cron every 4 hours

How do I make it always use the same token to authenticate and send the results by email?

**Comment by alejcas** [comment]
Hi if you can you should use a TokenBackend that uses a centralized store (like a db). From there you can always save and retrieve the most updated token.

If you use the default tokenbackend which uses the file system, then you should implement the mechanism provided by the `should_refresh_token()` method.

https://github.com/O365/python-o365/blob/master/O365/utils/token.py#L110
